apis/core/v1beta1/conditions.go

```diff
@@ -187,4 +195,37 @@ const (
 
 	// OpenStackControlPlaneNovaReadyErrorMessage
 	OpenStackControlPlaneNovaReadyErrorMessage = "OpenStackControlPlane Nova error occured %s"
+
+	// OpenStackControlPlaneClientReadyInitMessage
+	OpenStackControlPlaneClientReadyInitMessage = "OpenStackControlPlane Client not started"
+
+	// OpenStackControlPlaneClientReadyMessage
+	OpenStackControlPlaneClientReadyMessage = "OpenStackControlPlane Client completed"
+
+	// OpenStackControlPlaneClientReadyRunningMessage
+	OpenStackControlPlaneClientReadyRunningMessage = "OpenStackControlPlane Client in progress"
+
+	// OpenStackControlPlaneClientReadyErrorMessage
+	OpenStackControlPlaneClientReadyErrorMessage = "OpenStackControlPlane Client error occured %s"
+
+	// OpenStackClientReadyInitMessage
+	OpenStackClientReadyInitMessage = "OpenStack Client not started, waiting on keystone API"
+
+	// OpenStackClientKeystoneWaitingMessage
+	OpenStackClientKeystoneWaitingMessage = "OpenStack Client keystone ready"
```

Comment:
Since this constant is contains `Waiting` in its name, should the message have something other than `ready`?  Seems contradictory.

Reply:
"... waits for nothing" or the like?